### ISSUE TITLE
[2.0.0] Impossible to use as a module?

### ISSUE THREAD

**Issue by dpogue** [timruffles/ios-html5-drag-drop-shim#83]
Trying to pull the 2.0.0-beta.3 polyfill in to our project that uses TypeScript and WebPack2, and it seems effectively impossible to pull in as a module.

```typescript
import 'drag-drop-polyfill';
DragDropPolyfill.Initialize(); // Compiler Error:
// error TS2304: Cannot find name 'DragDropPolyfill'.
```

```typescript
import DragDropPolyfill from 'drag-drop-polyfill';
DragDropPolyfill.Initialize(); // Runtime Error:
// Uncaught TypeError: Cannot read property 'Initialize' of undefined
```

```typescript
import * as DragDropPolyfill from 'drag-drop-polyfill';
DragDropPolyfill.Initialize(); // Runtime Error:
// Uncaught TypeError: DragDropPolyfill.Initialize is not a function
```

What is the proper way to use this with WebPack/Browserify/Rollup?

**Comment by reppners** [comment]
UMD support is on the list but not yet implemented, as well as proper declaration file support on imports.

Personally I'm using it with SystemJS, the docs are off on the way to use it.

```TS
import "drag-drop-polyfill";

declare let DragDropPolyfill:any;

DragDropPolyfill.Initialize( {
    dragImageTranslateOverride: DragDropPolyfill.HandleDragImageTranslateOverride
} );
```

SystemJS takes care of loading files with global declarations properly. I don't know if and how WebPack is able to do this.

The plan is to migrate to TypeScript >2.0 which should result in proper UMD and declaration file support.

**Comment by reppners** [comment]
fixed by #100
